app/assets/stylesheets/nfg_ui/network_for_good/admin/nfg_theme/_forms.scss

```diff
@@ -64,7 +64,9 @@ label {
 // Sometimes rails likes to say a form control is valid even when it isn't... this helps ensure that
 // When simple_form (accurately) detects an error on the field...
 // that the invalid version wins ... avoiding false readings
-.is-invalid { border-color: $form-feedback-invalid-color !important; }
+.is-invalid { 
+  &, .custom-select{ border-color: $form-feedback-invalid-color !important; }
```

Comment:
@kylejmorin  you good with this update in this file? Resolving the issue shown on the attached image.
<img width="768" alt="screen shot 2019-01-24 at 11 58 57 am" src="https://user-images.githubusercontent.com/5702623/51694806-94674d00-1fcf-11e9-9f86-1c46a1de9ee1.png">


Fixed: 
<img width="653" alt="screen shot 2019-01-24 at 12 00 19 pm" src="https://user-images.githubusercontent.com/5702623/51694838-aa750d80-1fcf-11e9-905c-35a05ab264b3.png">

Reply:
@jonathanroehm, yes that looks good.